CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the "Liquibase" extension will be documented in this file
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.3
```

Comment:
I would argue that we should go to version `1.1.0`

This is according to [the VS Code guidelines](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions) which recommend it:

>  So, we recommend that extensions use major.EVEN_NUMBER.patch for release versions and major.ODD_NUMBER.patch for pre-release versions. For example: 0.2.* for release and 0.3.* for pre-release.